### ISSUE TITLE
Web interface, blank page ? 

### ISSUE THREAD

**Issue by CouinCouin** [ngrie/rtmpie#28]
Hi, 

I'm trying in a firt time on a VM, to test before eventually subscribe for a VPS, so for the moment, no https, no domain , just http.
During setup, I must set a domain, I tried a random domain name, and a tried the IP address of the VM, and in both case, I get a blank page when I browser _http://VM_IP/_ (the webserver anwsers without any delay, but the page is blank). 

I probably messed somethin, I tried also _http://VM_IP/admin_ but nothing better. 

How can I get the web interface please?

Thanks :)
Couin

**Comment by ngrie** [comment]
Try specifying the IP address as hostname during installation.

**Comment by CouinCouin** [comment]
Hi Niklas, 
Thanks for your answer, but I already tested this idea, withoout better result :(

**Comment by JustusFluegel** [comment]
check if caddy can connect to the php service, there should be some errors in the docker log if it can't. I was installing this on a kubernetes cluster and a blank page was exactly what i had when I messed up the php conf and it wasn't reachable. Alternatively  a simmilar error happens when either php can't connect to the database or the database isn't set up properly, for example i re-setup this container and a old volume from the database was still there, so the mysql password didn't get updated.

TLDR; take a look at `docker ps` and then inspect the logs of the containers listed there with `docker logs <name>`. If you can't solve the error yourself, maybe just post them here.

**Comment by CouinCouin** [comment]
Hi, thanks but I had never understood docker (how it runs, how to go inside to modify etc etc) and never found precise explanation of this, all pages I found was for people that already knows docker... useless for me so. 

Sorry, but I manually created my video server (also based on nginx + rtmp module for nginx).

This thread has no longer to be, I close it.
